modules/tournament/src/main/Schedule.scala

```diff
@@ -405,3 +439,61 @@ object Schedule:
           allPlannedSchedules = p :: allPlannedSchedules
           p :: newPlans
       .reverse
+
+  private[tournament] object ForTesting:
```

Comment:
could we move this to the test/ directory?

Reply:
OK, made an attempt -- can you take a look?